### PR TITLE
Add 'rhel8' suffix to cloud-event-proxy references to match comet repository name

### PR DESCRIPTION
**Pull request by nishant-parekh**
Changes in this PR:

    - update the image name in image references.
    - update the container name in linux PTP Daemon Daemon set CR.

Signed-off-by: Nishant Parekh <nparekh@redhat.com>

### CODE REVIEW

**Comment by nishant-parekh** [comment]
> we are trying a different approach to fix this.. instead of updating the image reference in the operator we will create a new delivery repository that does not have the rhel8 suffix.. and map the errata to release part of that release repository.. the request with the EXD Cloud Distribution team is being processed here: https://issues.redhat.com/browse/CLOUDDST-10106

Based on discussions with the release team engineer the policies for a layered image require the rhel8 suffix to be part of the delivery repository name.. so we would need to update the ptp operator reference of cloud-event-proxy.. these policies were not in effect when kube-rbac-proxy was provisioned so it is exempted from it..we will need this change to fix the issue we are seeing with the operator.

File: bindata/linuxptp/ptp-daemon.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
       priorityClassName: "system-node-critical"
       containers:
        {{ if (eq .EnableEventPublisher true) }}
-        - name: cloud-event-proxy
+        - name: cloud-event-proxy-rhel8
```
[the remark this answers is not in the document]
Comment (nishant-parekh) [reply bindata/linuxptp/ptp-daemon.yaml]: removed

File: manifests/image-references [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-kube-rbac-proxy:4.10
-  - name: cloud-event-proxy
+  - name: cloud-event-proxy-rhel8
```
[the remark this answers is not in the document]
Comment (SchSeba) [reply manifests/image-references:18]: do we also need the u/s image to follow this path?

[the remark this answers is not in the document]
Comment (nishant-parekh) [reply manifests/image-references:18]: no we do not need the upstream image to follow the same path/name this is specific to the delivery repo in comet and the naming policies associated with it.

**Comment by nishant-parekh** [comment]
Hi @SchSeba we got the release repository (without -rhel8 suffix) created and builds released part of the repo as well.. we are testing this fix out, if that works we would not need to change the image reference.
https://catalog.redhat.com/software/containers/openshift4/ose-cloud-event-proxy/617abc23c0d15aff4912ec0a?container-tabs=overview

**Comment by SchSeba** [comment]
Hi @nishant-parekh good to know they approved it

**Comment by SchSeba** [comment]
@nishant-parekh can you close this PR?

**Pull request closed without merging**